modules/videoio/include/opencv2/videoio.hpp

```diff
@@ -455,9 +461,12 @@ class CV_EXPORTS_W VideoCapture
     @param filename name of the opened video file (eg. video.avi) or image sequence (eg.
     img_%02d.jpg, which will read samples like img_00.jpg, img_01.jpg, img_02.jpg, ...)
 
+    @param apiPreference preferred Capture API to use. Can be used to enforce a specific reader
+    implementation if multiple are available: e.g. CV_CAP_FFMPEG or CV_CAP_IMAGES
+
     The methods first call VideoCapture::release to close the already opened file or camera.
      */
-    CV_WRAP virtual bool open(const String& filename);
+    CV_WRAP virtual bool open(const String& filename, int apiPreference = CAP_ANY);
```

Comment:
Seems nice to have such an overload function, but as far as I see this CAP_ANY is not known and the function doesn't handle it when it is used to call the basic function. Can you apply that also?